lib/arclight/custom_document.rb

```diff
@@ -21,6 +21,7 @@ class CustomDocument < SolrEad::Document
       t.function(path: 'archdesc/controlaccess/function/text()', index_as: %i[displayable facetable])
       t.occupation(path: 'archdesc/controlaccess/occupation/text()', index_as: %i[displayable facetable])
       t.places(path: 'archdesc/controlaccess/geogname/text()', index_as: %i[displayable facetable symbol])
+      t.places_coll(path: "archdesc[@level='collection']/controlaccess/geogname/text()", index_as: %i[displayable facetable symbol])
```

Comment:
I see that we are using `places_coll_ssim`. Do we still then need `displayable` and `facetable`?